History.markdown

```diff
@@ -1,6 +1,7 @@
 ## HEAD
 
   * Test against Jekyll 2 and 3. (#30)
+  * Lock to Listen 3.0.6
```

Comment:
yea, rails has a pattern for this on their repo: https://github.com/rails/rails/blob/master/actioncable/CHANGELOG.md

you just put a heading in when you do a release. when you haven't cut a release yet, you just insert changes at the top. then a release manager comes in and cuts a release by bumping the version, creating a tag, and inserting the version # into the changelog.